src/Compilers/Core/Portable/Syntax/SyntaxWalker.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace Microsoft.CodeAnalysis
 {
+#pragma warning disable RS0030 // Do not used banned APIs: SyntaxWalker
```

Comment:
>used [](http://example.com/codeflow?start=41&length=4)

Typo? "used" vs. "use"

Reply:
Changed in commit 5, note that around 50 other usages of this tense for similar comments in Roslyn that can be also changed.